lib/senkyoshi/models/questions/matching.rb

```diff
@@ -9,6 +9,18 @@ def initialize
       @distractors = []
     end
 
+    def strip_select_html(text)
+      tag_patterns = [/<\/?p[^>]*>/i,             # <p> tags
```

Comment:
I would suggest making a constant TAG_PATTERNS defined outside of the method. Will be a bit faster that way. Same goes for the union. Make a constant TAGS_RE for them. Since it is always going to be the same data.